acos_client/v30/axapi_http.py

```diff
@@ -64,6 +64,16 @@ def request(self, method, api_url, params={}, headers=None,
         LOG.debug("axapi_http: %s url = %s", method, api_url)
         LOG.debug("axapi_http: params = %s", json.dumps(params, indent=4))
 
+        # Update params with **kwargs for currently unsupported configuration
+        # of objects
+        formatted_kwargs = dict([(k.replace('_', '-'), v) for k, v in kwargs.iteritems()])
+        param_keys = params.keys()
+        if params != {}:
```

Comment:
I can do that, but I think
```python
acos.client.slb.virtual_port("vip1", "443", "https", template_client_ssl="foo")
```
looks better than
```python
acos.client.slb.virtual_port("vip1", "443", "https", axapi_args={"virtual_port": {"template_client_ssl": "foo"}})
```
and it would seem rare to me for someone to be using acos_client with no built in arguments. Though maybe with updates it would be common?